Source/WebCore/editing/FrameSelection.cpp

```diff
@@ -1839,7 +1841,11 @@ void CaretBase::invalidateCaretRect(Node* node, bool caretRectChanged)
 
 void FrameSelection::paintCaret(GraphicsContext& context, const LayoutPoint& paintOffset, const LayoutRect& clipRect)
 {
-    if (m_selection.isCaret() && m_caretPaint && m_selection.start().deprecatedNode())
+    bool caretPaint = true;
```

Comment:
`caretPaint` is a weird name and I don't know what it means. Does it mean "shouldPaintCaret"?